File: go/wam/7768.go
const WamGroupMemberUpdates = 7768 // channel: regular

type GroupMemberUpdatesEvent struct {
	FetchedMessageCount int64 `wam:"fetched_message_count"` // field 1
	GroupMemberUpdatesActionName GROUPMEMBERUPDATESACTIONNAME `wam:"group_member_updates_action_name"` // field 2
	GroupMemberUpdatesCurrentScreen GROUPMEMBERUPDATESCURRENTSCREEN `wam:"group_member_updates_current_screen"` // field 3
	GroupMemberUpdatesSessionId string `wam:"group_member_updates_session_id"` // field 4
	FetchedMessageLatency int64 `wam:"fetched_message_latency"` // field 5
}

File: rust/wam/7768.rs
pub const WAM_GROUP_MEMBER_UPDATES: u32 = 7768;

#[derive(Debug, Default)]
pub struct GroupMemberUpdatesEvent {
    /// field 1, wire `fetched_message_count`
    pub fetched_message_count: Option<i64>,
    /// field 2, wire `group_member_updates_action_name`
    pub group_member_updates_action_name: Option<GROUPMEMBERUPDATESACTIONNAME>,
    /// field 3, wire `group_member_updates_current_screen`
    pub group_member_updates_current_screen: Option<GROUPMEMBERUPDATESCURRENTSCREEN>,
    /// field 4, wire `group_member_updates_session_id`
    pub group_member_updates_session_id: Option<String>,
    /// field 5, wire `fetched_message_latency`
    pub fetched_message_latency: Option<i64>,
}